client/src/common/components/Confirmation.jsx

```diff
@@ -30,9 +36,9 @@ const Confirmation = ({ className, disabled, onCancel, onConfirm, title }) => (
 );
 
 Confirmation.propTypes = {
+  children: PropTypes.any,
```

Comment:
string or node? Or maybe:
```
children: PropTypes.oneOfType([
    PropTypes.string,
    PropTypes.node
  ]),
```

Reply:
Przemek suggested my last time to for .any for children proptype

Reply:
ok